train_openfold.py

```diff
@@ -272,6 +329,63 @@ def __init__(self, config):
         self.cached_weights = None
         self.last_lr_step = -1
 
+    def _compute_validation_metrics(self, 
```

Comment:
@christinaflo This ```_compute_validation_metrics``` is overwrites the parent class method so that it uses ```lddt_ca_multimer``` instead of ```lddt_ca```  and I turned ```all_atom_mask```, ```ca_pos``` and ```all_atom_mask_ca``` to None once they are not used anymore just to free up memory on GPU